### ISSUE TITLE
Security exception after last release

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#1413]
Reported by @97balakrishnan 
```
03-31 20:14:38.343 9842-9842/fr.free.nrw.commons.debug E/LocationServiceManager: Security exception
                                                                                 java.lang.SecurityException: "network" location provider requires ACCESS_COARSE_LOCATION or ACCESS_FINE_LOCATION permission.
                                                                                     at android.os.Parcel.readException(Parcel.java:1620)
                                                                                     at android.os.Parcel.readException(Parcel.java:1573)
                                                                                     at android.location.ILocationManager$Stub$Proxy.requestLocationUpdates(ILocationManager.java:606)
                                                                                     at android.location.LocationManager.requestLocationUpdates(LocationManager.java:880)
                                                                                     at android.location.LocationManager.requestLocationUpdates(LocationManager.java:464)
                                                                                     at fr.free.nrw.commons.location.LocationServiceManager.requestLocationUpdatesFromProvider(LocationServiceManager.java:106)
                                                                                     at fr.free.nrw.commons.location.LocationServiceManager.registerLocationManager(LocationServiceManager.java:94)
                                                                                     at fr.free.nrw.commons.nearby.NearbyActivity.onStart(NearbyActivity.java:259)
                                                                                     at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1260)
                                                                                     at android.app.Activity.performStart(Activity.java:6276)
                                                                                     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2389)
                                                                                     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2490)
                                                                                     at android.app.ActivityThread.-wrap11(ActivityThread.java)
                                                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354)
                                                                                     at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                     at android.os.Looper.loop(Looper.java:148)
                                                                                     at android.app.ActivityThread.main(ActivityThread.java:5443)
                                                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
                                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```
getting this exception on first time loading of nearby places after granting permission.
Nearby location doesnt load the first time due to this exception
Device Android 6.0.1
Version 2.7.0-debug master

**Comment by neslihanturan** [comment]
@97balakrishnan can you please expain this more briefly? I am not able to reproduce this so far.

**Comment by 97balakrishnan** [comment]
Yes,
**Steps to reproduce**
1. First time login
2. Click on notifications activity
3. Immediately click on home activity

![alt link](http://i66.tinypic.com/119xf1s.png)

**Comment by 97balakrishnan** [comment]
Steps to reproduce:
1. Log into the app
2. Open nearby places
3. Grant permission for location access

Now the nearby places does not load for me. The above exception was listed in my log cat.

If I got to home activity and return to nearby places it loads.

**Comment by neslihanturan** [comment]
Do you still facing with same issue? Or did it happen only once?

**Comment by 97balakrishnan** [comment]
No I can still produce this exception, multiple times

**Comment by neslihanturan** [comment]
Do you like to work on that to help us figure out what is going on, since you are the one can reproduce the issue?

**Comment by 97balakrishnan** [comment]
Yeah ok.. I will start working on this issue and get your guidance when required Thanks

**Comment by neslihanturan** [comment]
Sure, feel free to ask @97balakrishnan

**Comment by domdomegg** [comment]
Closed in #1440
